internal/pkg/manifest/workload_test.go

```diff
@@ -0,0 +1,221 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package manifest provides functionality to create Manifest files.
```

Comment:
nit: the package comment only needs to be in a single file (and not needed in test files)